examples/segmentation_3d/unet_evaluation_array.py

```diff
@@ -33,58 +32,57 @@ def main():
     config.print_config()
     logging.basicConfig(stream=sys.stdout, level=logging.INFO)
 
-    tempdir = tempfile.mkdtemp()
-    print(f"generating synthetic data to {tempdir} (this may take a while)")
-    for i in range(5):
-        im, seg = create_test_image_3d(128, 128, 128, num_seg_classes=1)
+    with tempfile.TemporaryDirectory() as tempdir:
```

Comment:
I think we should avoid this large block of indentation by adding the context from outside of main, something like:
```python
 if __name__ == "__main__":
    with tempfile.TemporaryDirectory() as tempdir:
        main(tempdir)
```

Reply:
good point, I will change it soon.
Thanks.